src/components/PageBanner/index.js

```diff
@@ -4,7 +4,7 @@ import ResponsiveImage from 'components/ResponsiveImage';
 import { FULL_WIDTH_RESPONSIVE_IMAGE_SIZES } from 'js/helpers/constants';
 
 const PageBanner = ({ imagesPath, imageFilename, imageExtension, imageTitle, headerText, extraClasses }) => (
-  <div className="coa-HeroHome__container">
+  <div className="coa-PageBanner__container">
```

Comment:
Would it make more sense to call it HeroPage or something similar to have consistency between component names? This kinda ties into a larger issue around the naming/structure of existing Janis components though...

Reply:
I was following the naming of the component itself. If anything, I think we should get rid of HeroHome and make PageBanner more flexible so we can use it in HeroHome.

Reply:
Made an issue to address more of this (https://github.com/cityofaustin/techstack/issues/2270) merging this PR now